ui/pages/settings/networks-tab/index.scss

```diff
@@ -291,34 +189,43 @@
   &__network-form-footer {
     display: flex;
     flex-flow: row nowrap;
-    padding: 0.75rem 0;
-    width: 95%;
-
-    @media screen and (max-width: $break-small) {
-      width: 93%;
-    }
+    padding: 12px 0;
 
     .btn--rounded {
       white-space: nowrap;
     }
 
     .btn-secondary {
-      margin-right: 0.375rem;
+      margin-left: auto;
     }
 
     .btn-primary {
-      margin-left: 0.375rem;
+      margin-left: 16px;
     }
 
     .btn-danger {
-      margin-right: 3.75rem;
+      margin-right: 16px;
+    }
+  }
+
+  &__add-network-form {
+    padding: 16px 24px;
+    grid-column: span 2; // spread both columns of grid layout
+
+    @media screen and (min-width: $break-large) {
+      max-width: 400px; // but only expand to 400px
+      padding: 16px;
+    }
+
+    &__alert {
+      margin-top: 0;
     }
   }
 
   &__add-network-form-footer {
     display: flex;
     flex-flow: row;
-    padding: 0.75rem 0;
+    padding: 12px 0;
     width: 60%;
 
     .btn-secondary {
```

Comment:
The main guts of this PR fixes a lot of arbitrary media queries and widths in favour of using CSS grid and our breakpoint sass variables